### DISCUSSION TITLE
Open Source?

### DISCUSSION THREAD

**Discussion by davebullock-ujet** [JudiniLabs/code-gpt-docs#106] [Q&A]
When are you planning on open sourcing the plugin?

> That repository is private for the moment, since the code is being prepared to be published as open source. I am working very hard to have a good path to receive contributions, it will soon be published on the https://codegpt.co page and in the official communities to start collaborating on the project. I appreciate your patience.

Why is it not open source?

**Comment by mariusheine** [comment]
@davila7 I would also like to know if this is possible for you? Because we (in our Company) would like to use the plugin and therefor contribute a bit, especially this issue here: https://github.com/davila7/code-gpt-docs/issues/153
